gnd/src/main/java/com/google/android/gnd/persistence/local/room/entity/FeatureMutationEntity.java

```diff
@@ -49,7 +50,7 @@
             entity = FeatureEntity.class,
             parentColumns = "id",
             childColumns = "feature_id",
-            onDelete = CASCADE),
+            onDelete = SET_DEFAULT),
```

Comment:
Removing `CASCADE` would lead to mutations in the db with no corresponding entities. Are we sure this is what we want? Two possible alternatives:

1. Mark features and observations as "DELETED" rather than actually removing them locally
2. Delete feature and observations, and write history of changes to a separate MUTATION_LOG table. 

IMO option 2 is preferable since it doesn't involve keeping entities in the db after they're deleted in order show the mutation log, and it doesn't rely on other components filtering out "deleted" items correctly. Wdyt?

Reply:
Agreed--I think option 2 is better. Let's do that!